UT4MasterServer/Services/TrustedGameServerService.cs

```diff
@@ -19,4 +19,24 @@ public TrustedGameServerService(DatabaseContext dbContext)
 		var cursor = await collection.FindAsync(x => x.ID == id, options);
 		return await cursor.SingleOrDefaultAsync();
 	}
+
+	public async Task UpdateAsync(TrustedGameServer server)
+	{
+		var options = new ReplaceOptions() { IsUpsert = true };
+		await collection.ReplaceOneAsync(x => x.ID == server.ID, server, options);
+	}
+
+	public async Task<List<TrustedGameServer>> ListAsync()
+	{
+		var result = await collection.FindAsync(x => true);
+		return await result.ToListAsync();
+	}
+
+	public async Task<bool?> RemoveAsync(EpicID id)
+	{
+		var result = await collection.DeleteOneAsync(x => x.ID == id);
+		if (!result.IsAcknowledged)
+			return null;
+		return result.DeletedCount > 0;
```

Comment:
Inconsistent in formatting (see `ClientService.RemoveAsync`)
```suggestion
		if (!result.IsAcknowledged)
			return null;

		return result.DeletedCount > 0;
```